docs/developers-guide/debug.md

```diff
@@ -56,7 +56,7 @@ index bb551950..a2b29beb 100644
 
 ```
 
-## Debugger instructions
+## Delve instructions
```

Comment:
I would leave this as "Debugger" (it's more generic). Recall we use gdb for the C code that is part of sysbox (e.g., nsexec).

Reply:
I did this to differentiate this section (delve specific) from the other one below (gdb specific). I can reword it as "Delve Debugger Instructions".